telemetry/producer.go

```diff
@@ -18,9 +18,9 @@ const (
 	Logger Dispatcher = "logger"
 )
 
-// BuildTopic produces records into the custom topic
-func BuildTopic(namespace string, record *Record) string {
-	return fmt.Sprintf("%s_%s", namespace, record.TxType)
+// BuildTopic creates a topic from a namespace and a recordName
+func BuildTopic(namespace, recordName string) string {
+	return fmt.Sprintf("%s_%s", namespace, recordName)
```

Comment:
Would renaming to `BuildTopicName` better convey that it is creating the name of a topic, not creating one?

Reply:
updated